styleguide/_themes/global/scss/bootstrap_variables.scss

```diff
@@ -7,28 +7,38 @@ $bootstrap-sass-asset-helper: false !default;
 //== Colors
 //
 //## Gray and brand colors for use across Bootstrap.
-
-$gray-base:              #10141a !default;
-$gray-darker:            #2e3238 !default;
-$gray-dark:              #4c5056 !default;
-$gray:                   #60646a !default;
-$gray-light:             #888c92 !default;
-$gray-lighter:           #c4c8ce !default;
-$gray-lightest:          #e2e6ec !default;
-
-$brand-primary:         #c40022 !default;
-$brand-success:         #7beb9a !default;
-$brand-info:            #c4c8ce !default;
-$brand-warning:         #ffe970 !default;
-$brand-danger:          #ffb16d !default;
+$white:                  #fff    !default;
+$black:                  #000    !default;
+$gray-base:              #666    !default;
+//bootstrap doesnt like leaving this out
+$gray:                   #666    !default;
+$gray-darker:            #4c4c4c !default;
+$gray-dark:              #333    !default;
+$gray-darkest:           #2a2a2a !default;
+$gray-light:             #b8b8b8 !default;
+$gray-lighter:           #dedede !default;
+$gray-lightest:          #efefef !default;
+
+//added for ordering issues
+$rackspace-red:     #c40022;
+$signup-green:      #5aaa28;
+$medium-teal:       #0dcfd9;
+$orange:            #f63;
+
+$brand-primary:         $rackspace-red  !default;
+$brand-secondary:       $medium-teal    !default;
+$brand-success:         $signup-green   !default;
+$brand-info:            #c4c8ce         !default;
```

Comment:
That is the default hex value provided by bootstrap. Since we don't have a $brand-info and arent using $brand-info anywhere I left it the default value bootstrap gave it. If we choose to add one in the future im all about utilizing a variable for it but as of now this is the only place its mentioned and this is the default value. I would remove it if bootstrap let me.